metricbeat/docs/modules/redis.asciidoc

```diff
@@ -63,9 +63,9 @@ metricbeat.modules:
   #maxconn: 10
 
   # Filters can be used to reduce the number of fields sent.
-  #filters:
+  #processors:
   #  - include_fields:
-  #      fields: ["stats"]
+  #      fields: ["beat", "metricset", "redis.info.stats"]
```

Comment:
Right. This is what I'm saying. That's why I'm considering the changes to metricbeat and processors order in the publisher. Maybe we also want to introduce some whitelisting (fields that can not be removed).